app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -13,7 +13,7 @@
   <FrameLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/question_player_background">
+    android:background="@color/question_player_fragment_background_color">
```

Comment:
All colors should follow this format:

`component_color_<activity-name>_<color-usage/type>_color`

For this it should be `component_color_question_player_activity_background_color`

Reply:
Got it